protocol/communities/manager.go

```diff
@@ -4813,3 +4826,7 @@ func (m *Manager) shareAcceptedRequestToJoinWithPrivilegedMembers(community *Com
 func (m *Manager) GetCommunityRequestsToJoinWithRevealedAddresses(communityID types.HexBytes) ([]*RequestToJoin, error) {
 	return m.persistence.GetCommunityRequestsToJoinWithRevealedAddresses(communityID)
 }
+
+func (m *Manager) SaveCommunity(community *Community) error {
```

Comment:
Minor thing, but i would be better to move it closer to other persistence wrappers: `SetPrivateKey`, `SaveCommunitySettings` and etc

Reply:
I think we just need to split manager.go :)